Clarify about distributed UPDATE/DELETE (#736)
timescale/docs: timescaledb/how-to-guides/hypertables/distributed-hypertables.md

perform the `JOIN` locally.

<highlight type="important">
In some cases, your processing speeds could be slower in a multi-node
cluster, because distributed hypertables need to push operations down
to the various data nodes. In operations like UPDATEs and DELETEs,
certain data needs to be first pulled from data nodes to the access

> node before sending further instructions down to the data nodes. It is
> important that you understand multi-node architecture before you begin,

Reviewer: ```suggestion
node before sending further instructions down to the data nodes. Make
sure you understand multi-node architecture before you begin,
```

Author: Ok, done.

> In some cases, your processing speeds could be slower in a multi-node
> cluster, because distributed hypertables need to push operations down
> to the various data nodes. In operations like UPDATEs and DELETEs,
> certain data needs to be first pulled from data nodes to the access
> node before sending further instructions down to the data nodes. Make
> sure you understand multi-node architecture before you begin, and plan
> your database according to your specific environment.

Reviewer: ```suggestion
In some cases, your processing speeds could be slower in a multi-node
cluster, because distributed hypertables need to push operations down
to the various data nodes. In operations like UPDATEs and DELETEs,
some data needs to be pulled from data nodes to the access node first,
before sending further instructions down to the data nodes. Make
sure you understand multi-node architecture before you begin, and plan
your database according to your specific environment.
```
